resources/app/src/router/people.js

```diff
@@ -7,6 +7,7 @@ const PersonEdit = () => import (/* person-detail */ '@/views/PersonEdit.vue')
 const PeopleList = () => import (/* people-index */ '@/views/PeopleList.vue')
 const OnboardingWizard = () => import (/* onboarding-wizard */ '@/views/OnboardingWizard.vue')
 const RequiredProfileUpdate = () => import (/* required-profile-update */ '@/views/RequiredProfileUpdate.vue')
+const DemographicsForm = () => import (/* person-detail */ '@/components/people/DemographicsForm.vue')
```

Comment:
I'm not sure how much of a difference this `person-detail` comment makes (I think it's a webpack "magic comment" for code-splitting), but it should probably be something relevant to the form like `demographics-form`.